.github/workflows/linux_intel_oneAPI.yml

```
@@ -0,0 +1,111 @@
+name: Linux tests (ifx + MKL)
```

I suggest changing this to:
```suggestion
name: Intel Fortran tests
```